libpyvinyl/Parameters/Collections.py

```diff
@@ -175,6 +177,7 @@ class MasterParameters(CalculatorParameters):
     additional ability to set values for the other parameters this master
     should control.
     """
+
```

Comment:
Add documentation for `__init__()` arguments.

Reply:
It is very likely to change with the support of the `pint` library. So I will postpone documentation to another PR.